ethcore/vm/src/action_params.rs

```diff
@@ -130,7 +130,7 @@ impl From<ethjson::vm::Transaction> for ActionParams {
 			gas: t.gas.into(),
 			gas_price: t.gas_price.into(),
 			value: ActionValue::Transfer(t.value.into()),
-			call_type: match address.is_zero() { true => CallType::None, false => CallType::Call },	// TODO @debris is this correct?
+			action_type: if address.is_zero() { ActionType::Create } else { ActionType::Call },
```

Comment:
I still believe that `ActionType::Create` here may not be correct. All `vmtests` are raw VM calls.